### DISCUSSION TITLE
[FEATURE] Add Gallery System

### DISCUSSION THREAD

**Discussion by FoggyMtnDrifter** [rocky-linux/rockylinux.org#254] [Ideas]
### Describe the solution you'd like

Our Community Team would love a place on the site to host galleries of images/videos from each of the events we attend. I envision it as follows:

The page URL should be located at /community/gallery. By default, galleries should be sorted by date and grouped by year. A nice-to-have would be search, and filters by year, event, location, etc.

Storage Requirements:
- Images should be hosted on an S3-compatible service.
- Metadata about events (year, name, url, etc.) should be stored in a flat file (JSON) locally in the project.

Media Requirements:
- Common image formats should be supported
- Embedding YouTube videos should be supported

### Additional Context

A future upgrade would be to query the S3 service for predetermined album in the JSON file and grab all the files that way to generate details, then cache the results.

**Comment by SanskariWolf** [comment]
I would like to work this issue. Can you please assign it to me.

**Comment by NebraskaCoder** [comment]
Issue still needs works.

**Comment by NebraskaCoder** [comment]
I moved this to a discussion until something is more actionable (assignee and requirements are discussed more).
